setup.cfg

```diff
@@ -37,6 +37,7 @@ packages = find:
 install_requires =
     typer==0.4.1
     pycurl==7.44.1
+    pycurl-requests==0.2.1
```

Comment:
Is this used somewhere, did not see it imported ?

Reply:
Forgot to delete it